app/controllers/application_controller.rb

```diff
@@ -12,13 +15,13 @@ class ApplicationController < Sinatra::Application
     set :session_secret, ENV.fetch('SESSION_SECRET') { SecureRandom.hex(64) }
     # Step 2: Enable Rack::Protection by uncommenting the following lines:
     #
-    # use Rack::Protection
+    use Rack::Protection
     # # `use Rack::Protection` automatically enables all modules except for the
     # # following, which have to be enabled explicitly
-    # use Rack::Protection::AuthenticityToken
-    # use Rack::Protection::EscapedParams
-    # use Rack::Protection::FormToken
-    # use Rack::Protection::RemoteReferrer
+    use Rack::Protection::AuthenticityToken
+    #use Rack::Protection::EscapedParams
```

Comment:
`Rack::Protection::EscapedParams` is missing here.  It provides more robust protection than just escaping text as we insert it into the HTML because it prevents us from storing malicious data